data/multiplayer/factions/drakes-aoh.cfg

```diff
@@ -4,7 +4,7 @@
     name= _"Drakes"
     image="units/drakes/flameheart.png"
     type=random
-    leader=Drake Flameheart,Inferno Drake,Drake Enforcer,Drake Warden,Drake Blademaster,Saurian Soothsayer,Saurian Oracle
+    leader=Drake Flameheart,Inferno Drake,Drake Enforcer,Drake Warden,Drake Blademaster,Saurian Warden,Saurian Seer,Saurian Prophet
     random_leader=Drake Flameheart,Inferno Drake,Drake Enforcer,Drake Warden,Drake Blademaster
     recruit=Drake Burner,Fire Drake,Drake Clasher,Drake Glider,Drake Fighter,Saurian Skirmisher,Saurian Augur,Drake Warrior,Drake Flare,Drake Thrasher,Sky Drake,Drake Arbiter,Saurian Ambusher,Saurian Soothsayer,Saurian Oracle
```

Comment:
you have forgotten to add Saurian Spearwoman as a recruit here, as stated in starting post of this PR

Reply:
I've already amended that, refresh the changes view.